### DISCUSSION TITLE
Several suggestions for ideas for developments

### DISCUSSION THREAD

**Discussion by rampionx** [IgnisDa/ryot#683] [Ideas]
Hi IgnisDa!

First of all, I want to thank you for your work, which is really great. I just installed your app and I have to say it's really nice. I'm currently looking for a self-hoested alternative to TV Time, and I think your project is perfect for me.

I hope I'm not making suggestions that have already been made but I haven't found. So here is what I would really like to see appear with RYOT:
    - A smartphone application to more easily follow series when you're half asleep on the couch. (And maybe, have push notification from it when a new media is available.)
    - The ability to choose the language of RYOT (The only point where I think I can help!).
    - The possibility of having the synopses in other languages
    - The option to have something like the season menu for anime and thus, update the progress in the same way.
    - To propose a Jellyseer/Ombi sync to make requests directly from RYOT.

Once again, thank you for this great work you've done, and I wish you good luck for the future!

**Comment by IgnisDa** [comment]
1) you can already install Ryot as a PWA
2) I've never implemented internationalisation. If you can, id be gratwful if you are willing to implement.
3) Yeah this is a good idea. Please create an issue for this.
4) the media providers we use for anime do not provide individual episode details.
5) can you explain more. I'm not familiar with either of the services.

Thank you for the kind words. You mentioned tvtime. Were you able to import data from that?

**Reply by rampionx** [reply]
1. Oh, ok. I didn't know this way of doing things. Thank you !
2.  I'm sorry, I wasn't very clear. I wanted to say that I can help translate the interface into my language if you open this possibility to your community. (French)
3. Ok, i'll do it.
4. I saw that you allow the use of anilist and MAL. I don't understand how Anilist works but MAL allows you to see titles, synopses, release dates, etc... ![image](https://github.com/IgnisDa/ryot/assets/10504691/4dea559c-fdd6-4371-88dc-631da6784e8d)
5. Jellyseer and Ombi are requesters. They allow plex/Jellyfin/Emby users to ask the server administrator to make media available. They can also be connected to Sonarr, Radarr, etc. which are automatic indexing and download management software.
So, with an integration, when we add media to the watchlist, we could automatically send a request to add the media to mediacenter. (I hope I was understandable)

Concerning TV Time, I have just made my request to recover my data. So far, I haven't imported anything. But I can get back to you when I have done so, if you wish.

**Reply by IgnisDa** [reply]
Yeah these sound doable. I'll take a look into them more closely when I have time.

Yes, do let me know about how your import goes.
